cmake/BuildLibCURL.cmake

```diff
@@ -62,15 +62,6 @@ macro(curl_build)
     endif()
     list(APPEND LIBCURL_CMAKE_FLAGS "-DENABLE_ARES=${ENABLE_ARES}")
 
-    # Setup http2 and nghttp2 library path
-    if(BUNDLED_LIBCURL_USE_NGHTTP2)
-        set(USE_NGHTTP2 "ON")
-        list(APPEND LIBCURL_FIND_ROOT_PATH ${NGHTTP2_INSTALL_DIR})
-    else()
-        set(USE_NGHTTP2 "OFF")
-    endif()
-    list(APPEND LIBCURL_CMAKE_FLAGS "-DUSE_NGHTTP2=${USE_NGHTTP2}")
```

Comment:
Nit: I would add explicit `-DUSE_NGHTTP2=OFF` (not it is disabled, because it is default).

I mean, replace this block with:

```cmake
# <..a comment re http2, tarantool 1.10 and 2.x..>
list(APPEND LIBCURL_CMAKE_FLAGS "-DUSE_NGHTTP2=OFF")
```